frontend/src/i18n/locales/nl.ts

```diff
@@ -91,6 +91,9 @@ export default {
     },
     subject: {
         register: "Registreer bij vak:",
+        ok: "Oké",
+        cancel: "Annuleer",
```

Comment:
```suggestion
        register: "Je bent uitgenodigd voor",
        ok: "Accepteer",
        cancel: "Annuleer",
```
Idem, mag ook iets gelijkaardigs zijn.